device/esp_tinyusb/tinyusb_msc.c

```diff
@@ -80,6 +80,8 @@ typedef struct {
     struct {
         msc_storage_obj_t *storage[TINYUSB_MSC_STORAGE_MAX_LUNS]; /*!< Storage objects */
         uint8_t lun_count;              /*!< Number of logical units (LUNs) supported by the storage. */
+        tusb_msc_callback_t event_cb;               /*!< Callback for mount changed events. */
+        void *event_arg;                            /*!< Argument to pass to the event callback. */
```

Comment:
If I understand it correctly, motivation for this change is public `tinyusb_msc_set_storage_callback()` function.

Could we reconsider existance of this function in esp_tinyusb 2.0? If it is not needed it can't create any problems...